birdhouse/config/canarie-api/docker_configuration.py.template

```diff
@@ -205,7 +213,7 @@ SERVICES = {
     'slicer': {
```

Comment:
@tlvu @huard 
I'd like validations about this item. 
The `stats` refer to `finch`, but the `monitoring` name is `Flyingpigeon`, and `finch` is already added elsewhere in the config.
I cannot find `slicer` in both of these birds' processes. 
Which process is being targeted so I can update the config accordingly?

Reply:
Since Finch is taking on more and more of Flyingpigeon, maybe the slicer functionality is now rightfully Finch?  David would be able to confirm.  Just my guess.